File: code/stable/GOOLTest/swift/PatternTest/Observer.swift
/** This is an arbitrary class acting as an Observer
*/
class Observer {
    var x: Int = 0
    
    init() {
        self.x = 5
    }
    
    func printNum() -> Void {
        print(self.x)
    }
    
    func getX() -> Int {
        return self.x
    }
    
    func setX(_ x: Int) -> Void {
        self.x = x
    }
}

File: code/stable/gooltest/python/PatternTest/Observer.py
## \brief This is an arbitrary class acting as an Observer
class Observer:
    def __init__(self):
        self.x = 5
    
    def printNum(self):
        print(self.x)
    
    def getX(self):
        return self.x
    
    def setX(self, x):
        self.x = x
